### PR TITLE
impr: get sticker pack

### LINKED ISSUE #508: [bug] `iwant` sticker pack file name may include illegal character

### LINKED ISSUE #509: [featue request] cache sticker pack tg file id
- 缓存 sticker pack 打包文件的id，再次以相同参数获取sticker pack时直接发送已上传的压缩包
- 添加 `nocache` 参数到 `iwant` 命令，该参数只在命令参数设置生效

### PR DESCRIPTION
**Pull request by hugefiver**
fix #508 
close #509

### CODE REVIEW

File: base/sticker.go [hunk 1]
```diff
@@ -660,19 +729,75 @@ loop:
...
 		File:     cpFile,
-	})
+	}, &tb.SendOptions{ReplyTo: ctx.Message(), AllowWithoutReply: true})
 	if errors.Is(err, tb.ErrTooLarge) {
 		if fileInfo != nil {
 			return ctx.Reply(fmt.Sprintf("太...太大了...有%.2fMB辣么大", float64(fileInfo.Size())/1024/1024))
 		}
 		return ctx.Reply("太大了，反正就是大")
 	}
+	if doc := respMsg.Document; doc != nil {
```
[the remark this answers is not in the document]
Comment (hugefiver) [reply base/sticker.go:744]: 也不一定 当send失败时doc也许大概可能没准应当是nil

File: base/sticker.go [hunk 2]
```diff
@@ -71,11 +98,11 @@ func (o stickerOpts) merge(m map[string]string, final bool) stickerOpts {
```
Comment (hugefiver) [inline base/sticker.go:98]: 应处理 `!pack`

**Review by hugefiver — COMMENTED** [review]
pause merge cause of some incoming fix

**Comment by hugefiver** [comment]
> pause merge cause of some incoming fix

done

**Review by icceey — APPROVED** [review]

**Review by icceey — APPROVED** [review]

**Pull request merged**